### DISCUSSION TITLE
AlDente 1.28.1

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1306] [General]
### All

- New App Icon!
- New Settings window which can better visualize AlDente's features and settings, especially on smaller screens. 
- New and improved popover window with tweaked design to better fit the macOS visual language.
- Removed the "Popup Animation" option due to the new popover window.
- Many smaller changes and fixes.

### Pro

- Users of Sailing Mode can now charge back to the charge limit by double-clicking the slider in the popup window.
- A warning icon in AlDente's status display is added if the app detects that the power supply is plugged in but cannot provide power.
- Added the ability to control the color of the Magsafe LED for MacBooks with MagSafe 3 via the Apple Shortcuts app.
- Added the ability to turn the Magsafe LED off while the device is asleep.

With peace & love from Vienna,
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.28.1'>AlDente 1.28.1</a>.</em>

**Comment by vincentpun** [comment]
Thank you for the release, and I have some feedback on the new design, and thank you for using a better macOS-like popover that is a lot closer to how all other menu extras behave.

I have some feedback on the design of the settings window:

- Taking the same approach of the controversial System Settings app is I would say a bit unexpected. I'm curious if screen space on small screens was the issue, whether adding a scrollable content area in the content pane would have helped. In fact, now a scrollable content area is pretty much mandatory for all panes because the height of the window has been drastically reduced.
- Scrollbar is always hidden even when I have them on always and impacts usability.
- The text next to the icons does not seem to line up vertically to me.
- I find the animations for the sidebar highlights a bit unnecessary and distracting. I can't think of another macOS app with sidebars that does this.
- Similarly I'm not sure why the corner radius of the window is much bigger compared to other apps.

**Reply by MatthiasKerbl** [reply]
Hi @vincentpun,

Thank you for your honest and constructive feedback. We will consider it for future development.

Best,
Matthias

**Comment by Swiss-Mac-User** [comment]
Love the updated & revised UI incl. rich Popover! In my opinion you achieved well to make AlDente more user friendly.

**Well done** 👏
